larray/core/array.py

```diff
@@ -3421,6 +3441,35 @@ def items(self, axes=None, ascending=True):
         """
         return SequenceZip((self.keys(axes, ascending=ascending), self.values(axes, ascending=ascending)))
 
+    # XXX: rename to iflat instead?
+    @lazy_attribute
+    def flat(self):
+        r"""Access the array by index as if it was flattened (all its axes were combined)
```

Comment:
must be added in api.rst

Reply:
Ok. I hesitated because I don't see any of our current users using this, but we never know...